docs/source/tutorials/nextflow.md

```diff
@@ -0,0 +1,303 @@
+# Tutorial: Running Nextflow in Python with Latch SDK
+
+Estimated time to complete: 30 minutes
+
+Nextflow is a popular framework for orchestrating bioinformatics workflows. In this tutorial, we will walk through how you can package an existing Nextflow script in Latch Python SDK!
+
+## Prerequisites
+
+Before we start, make sure you:
+
+* Install the [Latch SDK](../getting_started/quick_start.md).
+* Understand basic concepts of a workflow through our [Quickstart](../getting_started/quick_start.md) and [Authoring your Own Workflow](../getting_started/authoring_your_workflow.md).
+
+As an example, we will use Nextflow's RNA-seq pipeline example. Let's dive in!
+
+## Overview of the RNA-seq Nextflow Pipeline
+
+The [RNASeq-NF pipeline](https://www.nextflow.io/example4.html) maps a collection of read-pairs to a given reference genome and outputs the respective transcript model. The Nextflow team has kindly provided the GitHub repository to their full pipeline code [here](https://github.com/nextflow-io/rnaseq-nf).
+
+At a high level, RNASeq-NF uses the following components and tools:
+
+* Salmon 1.0.0
+* FastQC 0.11.5
+* Multiqc 1.5
+
+To run the Nextflow pipeline locally:
+
+```console
+# Clone the repository
+git clone https://github.com/nextflow-io/rnaseq-nf.git
+cd rnaseq-nf
+
+# Run the Nextflow script
+nextflow run main.nf
+```
+
+Once the execution finishes successfully, you will see a MultiQC report generated under `rnaseq-nf/results/multiqc_report.html`.
+
+## How to Wrap the Nextflow Pipeline in Latch SDK
+
+To port the above workflow to the Latch SDK, we have to complete three steps:
+
+1. Initialize a Latch workflow and clone the RNASeq-NF code inside the Latch workflow directory.
+2. Define your Dockerfile to install dependencies
+3. Define a workflow that executes the `nextflow run` command to run the Nextflow pipeline.
+
+### Step 1: Initialize the Latch workflow directory
+
+If you prefer to follow along, instead of creating the workflow from scratch, clone the final GitHub repository here:
+
+```console
+git clone https://github.com/latchbio/nextflow-latch-wf.git 
+```
+
+For those who prefer starting from scratch, first initialize a fresh Latch boiplerplate repository:
+
+```console
+latch init nextflow-latch-wf
+```
+
+Enter the workflow directory;
+
+```console
+cd nextflow-latch-wf
+```
+
+Clone the original RNASeq-NF code:
+
+```console
+https://github.com/nextflow-io/rnaseq-nf.git
+```
+
+Great! Now we have a boilerplate code to start modifying, as well as the Nextflow code that we will later use.
+
+### Step 2: Define your Dockerfile to install dependencies
+
+As the Latch workflow will be executed on a fresh machine on the Latch platform, we have to define a Dockerfile with the necessary dependencies for RNASeq-NF to run:
+
+![Nextflow Dockerfile](../assets/dockerfile-nextflow.png)
```

Comment:
would prefer to make these markdown code snippets and not have images so that ppl can copy/paste